lib/processCss.js

```diff
@@ -158,18 +161,18 @@ module.exports = function processCss(inputSource, inputMap, options, callback) {
 	var pipeline = postcss([
 		localByDefault({
 			mode: options.mode,
-			rewriteUrl: function(global, url) {
+			rewriteUrl: function(global, origUrl) {
 				if(parserOptions.url){
-					url = url.trim(" ");
+					var url = origUrl.trim(" ");
 
-					if(!loaderUtils.isUrlRequest(url, root)) {
-						return url;
+					if(!origUrl.replace(/\s/g, '').length || !loaderUtils.isUrlRequest(url, root)) {
+						return origUrl;
```

Comment:
@evilebottnawi I'm not 💯 on this yet, mainly because of the code noise, even if small 😛 is justifiable, compared to what we want to achieve here. `var origUrl` in particular, the `trim()` story, Can't we solve this otherwise ? Is the distinction between `url('') <=> url('  ')` really *really* worthwhile? Besides that not handling empty `@import/url()` of course which needs to land :D